.packit.yaml

```diff
@@ -2,98 +2,62 @@
 # See the documentation for more information:
 # https://packit.dev/docs/configuration/
 
-# NOTE: The Packit copr_build tasks help to check if every commit builds on
-# supported Fedora and CentOS Stream arches.
-# They do not block the current Cirrus-based workflow.
-
 downstream_package_name: skopeo
 upstream_tag_template: v{version}
 
-packages:
-  skopeo-fedora:
-    pkg_tool: fedpkg
-    specfile_path: rpm/skopeo.spec
-  skopeo-centos:
-    pkg_tool: centpkg
-    specfile_path: rpm/skopeo.spec
-  skopeo-eln:
-    specfile_path: rpm/skopeo.spec
+specfile_path: skopeo.spec
 
-srpm_build_deps:
-  - make
+# Disable automatic merging for Copr builds (and subsequent Testing Farm)
+merge_pr_in_ci: false
 
 jobs:
   - job: copr_build
     trigger: pull_request
-    packages: [skopeo-fedora]
-    notifications: &copr_build_failure_notification
+    notifications: &packit_failure_notification
       failure_comment:
-        message: "Ephemeral COPR build failed. @containers/packit-build please check."
-    targets:
-      - fedora-all-x86_64
-      - fedora-all-aarch64
+        message: "Packit jobs failed. @containers/packit-build please check."
     enable_net: true
+    targets:
+      - epel-10-x86_64
+      - epel-10-aarch64
+    actions:
+      post-upstream-clone: "curl -O https://gitlab.com/redhat/centos-stream/rpms/skopeo/-/raw/c10s/skopeo.spec"
```

Comment:
this is missing `--fail` otherwise curl won't error for 4xx/5xx http errors. I guess the build would then of course fail but it would be harder to spot?

Reply:
done